src/Calendar.js

```diff
@@ -221,6 +221,20 @@ const Calendar = ({
     return allDays;
   };
 
+  function onDaySelect(date) {
+    dispatchSelectedDays({ type: 'SELECT', date });
```

Comment:
You're passing `isStandard` argument to `onDaySelect`, but it hasn't used in the function. By the way, I think maybe using ES6 syntax would be better for these functions.

Reply:
Yeah, it's so much better. Except that eslint complains about returning nothing in an arrow function :smile: 
Also, about the isStandard, I passed it to every handler function to keep the signature consistent between them.

Reply:
Why would it complain? Just replaced them with ES6 arrow functions with no problem. Maybe you have a global config for ESLint. Make sure you run `yarn lint` to check style guides. I think `isStandard` can be passed along with date to `onDaySelect` and alike functions like this:
```
onDaySelect({ ...dayItem, isStandard });
onDayEnter({ ...dayItem, isStandard });
```
and then we don't have to define `isStandard` parameter in `onDayEnter`.